### ISSUE TITLE
Google device access login button doesnt work

### ISSUE THREAD

**Issue by thomas-svrts** [aegjoyce/ha-addons#42]
For some reason I cant configure the ‘google devices’ plugin.
After entering project id and oAuth details I should press the login button which redirect you to a google login prompt. But the button doesnt work.
Can this be related to the ingress config?

![00C8E47B-00B6-4B66-A1E8-E37B4634F2E2](https://user-images.githubusercontent.com/81048302/215320708-712fe77a-6134-4e04-ae29-a002e071947c.jpeg)

**Comment by aegjoyce** [comment]
Have you installed the cloud plugin? It's a prerequisite for the google devices plugin and needs to be installed and set up first before you can try using the google device plugin.

**Comment by thomas-svrts** [comment]
Ok forgot that.

but also that doesnt work very well. 
i.e login via google button doesnt work.
When Login via appleID I ended on this page…
![E6075F40-DE82-4977-A111-314181D8597A](https://user-images.githubusercontent.com/81048302/215324144-70f593ae-f577-45c5-b9b3-48c096f369cb.png)

**Comment by aegjoyce** [comment]
Hmm. I'm new to Ingress so it's probably going to take me a frustratingly long time to fix. However there's an easy workaround - log in to Scrypted through the webui (your_ha_ip_address:10443) and set up cloud and google devices through this instead of through ingress. Have just tried and it worked fine with apple

**Comment by aegjoyce** [comment]
Closing for now with the comment above as accepted workaround, happy to reopen if causing significant issues

**Comment by townsmcp** [comment]
@aegjoyce i have installed the HA addon and tried to go through setting up the Scrypted Cloud plugin. I have changed the port forward mode to custom domain but that doesn’t seem to work (I tried to enter the Nabu Casa address). I then tried adding :10443 after the Nabu Casa address etc. when I go to Advanced and click Register nothing happens. I tried to access the console via my local ip (https://192.168.1.198:10443) and I get into Scrypted and can’t register still. Any other suggestions?
